mlflow/ml-package-versions.yml

```diff
@@ -651,13 +651,15 @@ langchain:
         ]
     run: |
       pytest tests/langchain/test_langchain_model_export.py
+      pytest tests/langchain/test_langchain_databricks_dependency_extraction.py
```

Comment:
Not related to this PR, but this test file was missing in cross version test so adding back.